.changesets/feat_geal_entity_cache_metrics.md

```diff
@@ -0,0 +1,43 @@
+### Entity cache: convert spans and metrics to the new format ([PR #5625](https://github.com/apollographql/router/pull/5625))
+
+<!-- [ROUTER-387] -->
+This converts the entity cache metrics to the new metric format and using the custom telemetry stack.
+
+Here is an example of configuration:
+
+```yaml
```

Comment:
Can we add this recipe to the Entity Cache docs page?

cc @shorgi

Reply:
Ideally these super helpful recipes live in the docs where most customers are reading about these features as opposed to getting lost in release notes!

Reply:
Absolutely, this should be in the documentation.  Perhaps this could be added to https://github.com/apollographql/router/pull/5574, which is currently in progress?

Reply:
The WIP docs of https://github.com/apollographql/router/pull/5574 already have example configurations.

Reply:
It has already been added